CoseHandler/CoseHandler.cs

```diff
@@ -557,14 +558,14 @@ public static class CoseHandler
         {
             errorCodes.Add(ValidationFailureCode.SigningCertificateUnreadable);
             content = null;
-            return new ValidationResult(false, errorCodes);
+            return new ValidationResult(false, errorCodes, validationType: ContentValidationType.ContentValidationNotPerformed);

```

Comment:
Does this mean that we're not validating content by default? Or is that just the way it's worded?

Reply:
Not exactly, we always attempt to do content validation as long as 1.) the message is well formed and 2.) the signature is "trusted". This isn't a logic change, just adding more info for verbose calls

Reply:
I can definitely change the enum name to something more accurate if you have any ideas?